web/pdf_thumbnail_view.js

```diff
@@ -199,6 +199,14 @@ var PDFThumbnailView = (function PDFThumbnailViewClosure() {
       if (!noCtxScale && outputScale.scaled) {
         ctx.scale(outputScale.sx, outputScale.sy);
       }
+      this.image  = document.createElement('img');
+      this.image.id = this.renderingId;
+      this.image.style.height = canvas.style.height;
+      this.image.style.width = canvas.style.width;
+      this.image.className = 'thumbnailImage';
+      this.image.setAttribute('aria-label', mozL10n.get('thumb_page_canvas',
+            {page: this.id}, 'Thumbnail of Page {{page}}'));
```

Comment:
Indent this line with only two spaces instead of six. Also add spaces before `}` and after `{` to get `{ page: this.id }`.